api/resource/definitions/perf/perf.proto

```diff
@@ -81,4 +81,3 @@ message MemorySpec {
   uint64 direct_map2m = 47;
   uint64 direct_map1g = 48;
 }
-
```

Comment:
these changes shouldn't be there

Reply:
I didn't modify any of those files so I'm not sure why they are.